docs/api.md

```diff
@@ -406,7 +406,7 @@ You will then need to call [`puppeteer.connect([options])`](#puppeteerconnectopt
 Puppeteer looks for certain [environment variables](https://en.wikipedia.org/wiki/Environment_variable) to aid its operations.
 If Puppeteer doesn't find them in the environment during the installation step, a lowercased variant of these variables will be used from the [npm config](https://docs.npmjs.com/cli/config).
 
-- `HTTP_PROXY`, `HTTPS_PROXY`, `NO_PROXY` - defines HTTP proxy settings that are used to download and run Chromium.
+- `HTTP_PROXY`, `HTTPS_PROXY`, `ALL_PROXY`, `NO_PROXY` - defines HTTP proxy settings that are used to download and run Chromium.
```

Comment:
To show that's a valid variable to be useful. I first tried `SOCKS_PROXY` variable but that is not a standard variable.

Reply:
Ah ok, I see. Didn't know about the standard :)

Reply:
So maybe the text should also be updated - currently it says ...defines HTTP proxy settings...